### ISSUE TITLE
TS2339 error mismatch between tss and tsc

### ISSUE THREAD

**Issue by dvogel** [clausreinke/typescript-tools#58]
Given this file:

```typescript
['a', 'b', 'c'].find(function(letter, idx, ary){
  return letter == 'c';
});
```

I can run it through `tsc` 1.5.0-beta without any errors.

However, when using typescript-tools sha 619d879a673e210cf4ad97a40e16421db9ac013e I see an error when I run the file through `tss`:

```bash
echo showErrors | tss array_of_string_find_method.ts | jq '.'
"loaded /tmp/array_of_string_find_method.ts, TSS listening.."
[
  {
    "file": "/tmp/array_of_string_find_method.ts",
    "start": {
      "line": 1,
      "character": 17
    },
    "end": {
      "line": 1,
      "character": 21
    },
    "text": "Property 'find' does not exist on type 'string[]'.",
    "code": 2339,
    "phase": "Semantics",
    "category": "Error"
  }
]
```

**Comment by clausreinke** [comment]
> I can run it through  tsc  1.5.0-beta without any errors.

I doubt that!-) tsc tends to generate code even in the presence of non-fatal errors, but it does emit the error you mentioned. `find` is an ES6/2015 method, and tsc/tss default to target ES5, so there has to be an error. Adding `--target es6` eliminates the error in both cases.

**Comment by dvogel** [comment]
My mistake. I didn't realize that adding filenames to the `tss` command line overrode the default behavior of loading the `tsconfig.json` file. I used the `--target ES6` parameter on the command line when testing `tsc`. At that point I had not yet read this on the typescript wiki:

> When input files are specified on the command line, tsconfig.json files are ignored.

Thanks for your explanation.
